src/dashboard/store/modules/settings.js

```diff
@@ -0,0 +1,25 @@
+import browser from 'webextension-polyfill';
+
+export default {
+  namespaced: true,
+
+  state: {
+    totalAmount: 10,
+  },
+
+  getters: {},
+
+  mutations: {
+    updateSettings(state, settings) {
+      Object.assign(state, settings);
+      browser.storage.local.set({ settings: state });
```

Comment:
Isn't mutations just for updating the state variable? I.e. shouldn't this be a method?

Reply:
For mutating *any* state, including the state variable.